### ISSUE TITLE
Error on installation

### ISSUE THREAD

**Issue by wmushtaq** [livewire/livewire#519]
I am getting the following error upon installation of package

```
In LivewireServiceProvider.php line 30:
  syntax error, unexpected '}', expecting identifier (T_STRING) or function (T_FUNCTION) or const (T_CONST)
```

I am using Laravel 5.7 on PHP 7.1.33

**Comment by cbaconnier** [comment]
Because the import has [a comma at the end of the use declaration](https://github.com/livewire/livewire/blob/master/src/LivewireServiceProvider.php#L30) and it's a [feature introduced with PHP 7.2](https://wiki.php.net/rfc/list-syntax-trailing-commas)

Note: PHP 7.1 is not supported anymore https://www.php.net/supported-versions.php

**Comment by wmushtaq** [comment]
Thanks @cbaconnier

**Comment by calebporzio** [comment]
@wmushtaq - I don't think there are any real reasons we need to break support for 7.1.

Would you be open to PRing a fix to make Livewire compatible with 7.1 and below?
